### PR TITLE
Add globalization

### PR DESCRIPTION
**Pull request by loay**
Connect to strongloop/loopback/issues/2422

### CODE REVIEW

**Comment by loay** [comment]
test please

**Comment by loay** [comment]
test please

**Comment by 0candy** [comment]
@loay Can you update to use the latest format? See https://github.com/strongloop/loopback-connector-dashdb/pull/3

**Comment by qpresley** [comment]
test please

**Comment by qpresley** [comment]
test please

**Comment by loay** [comment]
test please

File: lib/transaction.js [hunk 1]
```diff
@@ -27,7 +28,8 @@ function mixinTransaction(DB2, db2) {
 
     if (isolationLevel !== Transaction.READ_COMMITTED &&
       isolationLevel !== Transaction.SERIALIZABLE) {
-      var err = new Error('Invalid isolationLevel: ' + isolationLevel);
+      var err = new Error(g.f('Invalid {{isolationLevel}}: %s'),
+      isolationLevel);
```
Comment (0candy) [inline lib/transaction.js]: This should be indented and end bracket in the wrong place

File: package.json [hunk 2]
```diff
@@ -20,7 +20,9 @@
     "async": "^1.5.0",
     "debug": "^2.2.0",
     "loopback-connector": "^2.3.0",
-    "loopback-ibmdb": "^1.0.0"
+    "loopback-ibmdb": "^1.0.0",
+    "path": "^0.12.7",
```
Comment (0candy) [inline package.json]: Don't need path.

File: package.json [hunk 3]
```diff
@@ -20,7 +20,9 @@
...
+    "strong-globalize": "^2.7.0"
```
Comment (0candy) [inline package.json]: Use latest version 2.8.0

File: lib/globalize.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+'use strict';
```
Comment (0candy) [inline lib/globalize.js]: Please add license text to beginning of file.
```
// Copyright IBM Corp. 2016. All Rights Reserved.
// Node module: loopback-connector-db2
// This file is licensed under the Artistic License 2.0.
// License text available at https://opensource.org/licenses/Artistic-2.0
```

File: lib/discovery.js [hunk 5]
```diff
@@ -196,7 +197,7 @@ function mixinDiscovery(DB2, db2) {
     //   options = {};
     // }
     if (typeof options !== 'object') {
-      throw new Error('options must be an object: ' + options);
+      throw new Error(g.f('options must be an {{object}}: %s'), options);
```
Comment (0candy) [inline lib/discovery.js]: Should be `throw new Error(g.f('options must be an {{object}}: %s', options));`
End bracket in the wrong place.

File: lib/migration.js [hunk 6]
```diff
@@ -36,7 +37,7 @@ module.exports = function(DB2) {
     async.each(models, function(model, done) {
       if (!(model in self._models)) {
         return process.nextTick(function() {
-          done(new Error('Model not found: ' + model));
+          done(new Error(g.f('{{Model}} not found: %s'), model));
```
Comment (0candy) [inline lib/migration.js]: Same here, end bracket in the wrong place.

**Comment by loay** [comment]
yes the eager load connector is not related to globalization

**Comment by 0candy** [comment]
If the test are not needed, then please remove that commit before squashing and merging the PR.

**Comment by loay** [comment]
@qpresley @rmg 
I am ready to squash and merge. Do you guys need this PR for further testing? Thanks

**Comment by qpresley** [comment]
@loay I would like to see if the CIS tests start to pass .. if they fail you can still merge.

**Pull request merged**